src/applications/personalization/profile/components/personal-information/PersonalInformationSection.jsx

```diff
@@ -105,10 +106,8 @@ const PersonalInformationSection = ({
             <span className="vads-u-font-weight--bold vads-u-display--block">
               If you receive VA benefits, but aren’t enrolled in VA health care
             </span>
-            Call us at <va-telephone contact="800-827-1000" /> (
-            <a href="tel:711" aria-label="TTY: 7 1 1.">
-              TTY: +711
-            </a>
+            Call us at <va-telephone contact="8008271000" /> ( TTY:{' '}
+            <va-telephone contact={CONTACTS['711']} />
```

Comment:
So yea anywhere the the TTY type number is used it should be something like: 

`(<va-telephone contact={CONTACTS['711']} tty />)`

So that the text 'TTY' appears _inside_ the clickable link text and that prop will also ensure the aria label include the TTY text as well.

Here is what the component should look like when rendered
![Screen Shot 2022-12-01 at 12 42 42 PM](https://user-images.githubusercontent.com/8332986/205145035-e27918c5-29f6-43de-89af-9ab5ba1f07ca.png)